jackson/src/test/java/com/lightbend/lagom/serialization/JacksonSerializerFactoryTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
```

Comment:
Unfortunately, this branch has fallen out of step with some of the build changes on `master`, so this header isn't passing the pull-request validation (which builds a merge commit with this branch + `master`).

It's looking for an extra newline after the comment. Simplest thing to do might be to merge `master` to this branch (or rebase this branch) and then run `sbt test:headerCreate`.

Reply:
I merged with master, hopefully everything will work now